package.json

```diff
@@ -44,7 +45,8 @@
     "pino-pretty": "^5.1.3",
     "retry": "^0.13.1",
     "tmp": "^0.2.1",
-    "xmlrpc": "^1.3.2"
+    "xmlrpc": "^1.3.2",
+    "zod": "^3.17.10"
```

Comment:
While zod provides nice integration with typescript I doubt that it was reasonable to introduce it here.
Server project already has `ajv` validator. 
It is is used in 1 case, but introduces dependency. 
I am still doubt that it was wise step.

Reply:
No need. I am OK to use both. They cover different needs. Ajs is used to check for messages against well defined retemote schema, without actual interpretation. While Zod is interface definition, that we use in  code.